scripts/update-frontend-canister.sh

```diff
@@ -8,7 +8,20 @@ die() {
 
 cargo --version >/dev/null || die "Must have cargo installed."
 
-export RUSTFLAGS="--remap-path-prefix=\"${PWD}\"=./ --remap-path-prefix=\"${HOME}\"=_/"
+SCRIPT=$(readlink -f "${0}")
+SCRIPT_DIR=$(dirname "${SCRIPT}")
+cd ${SCRIPT_DIR}/..
+
+if [ -z "${CARGO_HOME}" ]
+then
+  export CARGO_HOME="${HOME}/.cargo"
+fi
```

Comment:
This is already the default.  Am I missing something, or does this have no effect?

Reply:
It sets the environment variable to the default value if the environment variable is undefined (`-z "${CARGO_HOME}"`).

Reply:
Right, but according to https://doc.rust-lang.org/cargo/reference/environment-variables.html, that's the default.  Setting it here makes the reader wonder why it's set.  I'd recommend not to set it, and let cargo set its own defaults.
```suggestion
```

Reply:
It's not set for the sake of cargo, but for the sake of remapping path prefixes in `RUSTFLAGS`.